src/components/Modal.js

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+const Modal = ({ children, isOpen, toggle }) => {
+  return (
+    <div>
```

Comment:
You could use `<>` fragment instead of a div parent
```suggestion
    <>
```

Reply:
I was using them, but another components was using div, so i use it to standarize it

Reply:
In this case is OK a div as we can add an id later to test, so is valid

CC: @denisosuna

Reply:
Most of the times <> are when you don't need tests for the return value.